tests.sh

```diff
@@ -2,7 +2,7 @@
 
 step=1
 total_steps=3
-max_retries=10
+max_retries=120 #seconds for the server reachability timeout
```

Comment:
Is this truly needed? I wonder if we should wait that long.

Reply:
I'm going to just keep the comment